imaginate_api/utils.py

```diff
@@ -98,3 +99,15 @@ def build_image_from_url(url):
     filename=str(url).rstrip("/").split("/")[-1],
     content_type=type_photo,
   )
+
+
+# helper function that returns a timestamp as an integer, no matter if day is inputed as a timestamp or day #
+# WARNING, do not enter a timestamp from BEFORE august 1st 2024.
+def calculate_date(day: str | int):
```

Comment:
This now takes in a string OR integer, calculates a timestamp and returns it as an integer